src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1574,8 +1509,131 @@ private protected override void CheckLabel(RoleMappedData examples, out int weig
             weightSetCount = 1;
         }
 
-        protected override BinaryPredictionTransformer<TScalarPredictor> MakeTransformer(TScalarPredictor model, Schema trainSchema)
-            => new BinaryPredictionTransformer<TScalarPredictor>(Host, model, trainSchema, FeatureColumn.Name);
+        protected override BinaryPredictionTransformer<TModelParameters> MakeTransformer(TModelParameters model, Schema trainSchema)
+            => new BinaryPredictionTransformer<TModelParameters>(Host, model, trainSchema, FeatureColumn.Name);
+    }
+
+    public sealed class SdcaCalibratedBinaryTrainer :
```

Comment:
>public [](start = 4, length = 6)

internal

Reply:
Ok.

---
In reply to: [256257461](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256257461) [](ancestors = 256257461)